domains/my-portfolio.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "JesusMiguel03",
+    "email": "jmdefreitas10@gmail.com"
+  },
+  "records": {
+    "CNAME": "https://portfolio-git-master-sircels-projects.vercel.app/"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.